find-index.js

```diff
@@ -1,11 +1,10 @@
 export const findIndex = (arr, callback) => {
-  let newArr = [];
+  // let newArr = [];
 
   for (let index = 0; index < arr.length; index++) {
     const item = callback(arr[index]);
     if(item) {
-      newArr = [...newArr, index];
+      return [index];
     }
```

Comment:
Nice work on the findIndex, Sofia.  I did take off partial credit for this, bc the findIndex array has one more gotchya.  

```Returns the index of the found item, -1 if no item is found.``` 

So for example, if your array below was ```[1, 2, 3, 4]``` but your callback was something like this ```const callback = item => item > 5;``` it should return ```-1```

I find it helpful to play with [this example in MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex). 

Again, don't worry about fixing this since it's past submission date unless you want to work on it.